easycv/models/classification/classification.py

```diff
@@ -274,7 +241,7 @@ def update_extract_list(self, key):
     def forward(
         self,
         img: torch.Tensor,
-        mode: str = 'train',
+        mode: str = 'test',
```

Comment:
why modify to `test`, default should be `train`

Reply:
default mode used to extract feature,  so may be  extract_feature?